### ISSUE TITLE
Notifications for copilots for approved projects

### ISSUE THREAD

**Issue by gondzo** [topcoder-platform/tc-notifications#95]
We used to send notifications to all copilots about new projects that are approved, but now that we have a copilot manager, maybe we shouldn't create those notifications? Copilots can't see the project page without joining anyway, so better to remove these?
https://github.com/topcoder-platform/tc-notifications/blob/8645140cad9690880409891bd1c621addf50abfb/connect/events-config.js#L61

**Comment by vikasrohit** [comment]
Yes, seems logical to me. Waiting for product team and copilot managers to give green signal.
